components/automate-ui/src/app/pages/project/rules/project-rules.component.ts

```diff
@@ -131,9 +143,10 @@ export class ProjectRulesComponent implements OnInit, OnDestroy {
 
   createCondition(attribute = '', operator = '', values = ''): FormGroup {
     return this.fb.group({
+      // Must stay in sync with error checks in project-rules.component.html
       attribute: [attribute, Validators.required],
       operator: [operator, Validators.required],
-      values: [values, Validators.required]
+      values: [values, [Validators.required, Validators.pattern(Regex.patterns.NON_BLANK)]]
```

Comment:
All input fields should have both `required` and `pattern`/`NON_BLANK` validators. One would think that a pattern should cover both cases, but no, for some reason it does not, even if the pattern explicitly disallows an empty value.